cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,83 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
-describe('Settings page', () => {
-  before(() => {
+import SettingsPageObject from '../support/pages/setting.pageObject';
+import HomePageObject from '../support/pages/home.pageObject';
+import ProfilePageObject from '../support/pages/profile.pageObject';
+import SignInPageObject from '../support/pages/signIn.pageObject';
 
-  });
+const settingsPage = new SettingsPageObject();
+const homePage = new HomePageObject();
+const profilePage = new ProfilePageObject();
+const signInPage = new SignInPageObject();
 
-  beforeEach(() => {
+describe('Settings page', () => {
+  let user;
 
+  beforeEach(() => {
+    cy.task('db:clear');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
   });
 
   it('should provide an ability to update username', () => {
+    signInPage.login(user.email, user.username, user.password);
+    settingsPage.visit();
+    settingsPage.usernameField.clear();
```

Comment:
```suggestion
    settingsPage.usernameField.clear();
```
move this method into typeNewUsername()